src/pages/Home.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import HomeArticle from "../components/HomeArticle";
+import "./Home.css";
+
+function Home() {
+  const [articles, setArticles] = useState([]);
+
+  async function getArticles() {
```

Comment:
Ideally we should have our api calls in their own file to keep our components tidy and allow our api calls to be reusable if we need them again.